00_core.ipynb

```diff
@@ -39,7 +39,7 @@
     "import os\n",
     "\n",
     "base_folder = os.environ.get(\n",
-    "    \"S4REFSIMTOOL_INPUT\", \"/global/cscratch1/sd/keskital/s4sim/reference_tool/out_v1/\"\n",
+    "    \"S4REFSIMTOOL_INPUT\", \"/project/projectdirs/cmbs4/dm/dstool/input/\"\n",
```

Comment:
The `/project/projectdirs` alias has limited lifespan. It is best to start moving to replacing it with `/global/cfs/cdirs`.

Reply:
sure thanks